governance/roles.md

```diff
@@ -145,8 +145,29 @@ and led by a Security Assessment Facilitator, who will:
   of significant process changes).
 * triage issues related to security assessments.
 
-
-### Triage Team
+### Meeting facilitator
+
+The group meetings are an important part of community building and the 
+facilitator ensures a welcoming and inclusive atmosphere. In keeping with these
+goals, the meeting facilitator has the following responsibilities:
+* prepares the meeting notes with template and agenda
```

Comment:
Considering that a non-chair member can have a `Meeting facilitator` role (as described below) there is duplication with the chair role: 
![image](https://user-images.githubusercontent.com/16246778/63592273-e0a45b00-c5b9-11e9-8279-bbb21fcff49c.png)

Maybe it makes sense to remove that responsibility from the Chair members and give it to the Meeting facilitators?